python/tests/integration/arcticdb/version_store/test_snapshot.py

```diff
@@ -828,19 +815,34 @@ def test_delete_snapshot_on_updated_and_appended_dataframe(basic_store_tiny_segm
     lib.delete_version(symbol1,1)
     lib.delete_version(symbol1,2)
 
-    assert df_1.equals(lib.read(symbol1).data)
-    assert df_updated.equals(lib.read(symbol1, as_of=snap1).data)
-    assert df_final.equals(lib.read(symbol1, as_of=snap2).data)
-    assert lib.list_versions(symbol1)[0]["deleted"]    
-    assert lib.list_versions(symbol1)[1]["deleted"]    
-    assert not lib.list_versions(symbol1)[2]["deleted"]    
+    assert_frame_equal(df_1, lib.read(symbol1).data)      
+    assert_frame_equal(df_updated, lib.read(symbol1, as_of=snap1).data)      
+    assert_frame_equal(df_final, lib.read(symbol1, as_of=snap2).data)      
+    assert [ver["deleted"] for ver in lib.list_versions(symbol1)] == [True, True, False]
 
     lib.delete_snapshot(snap1)
     lib.delete_snapshot(snap2)
 
-    assert df_1.equals(lib.read(symbol1).data)
+    assert_frame_equal(df_1, lib.read(symbol1).data)      
     assert len(lib.list_versions(symbol1)) == 1
     with pytest.raises(NoDataFoundException):
         lib.read(symbol1, as_of=snap1).data
     with pytest.raises(NoDataFoundException):
         lib.read(symbol1, as_of=snap2).data
+
+
+df1 = create_df_index_rownum(2, 0, 2)
```

Comment:
Looks like you forgot some test code here

Reply:
removing